pesign-obs-integration.spec

```diff
@@ -17,6 +17,10 @@
 # needssslcertforbuild
 
 
+%if (0%{?fedora})
```

Comment:
is the extra `()` necessary? it is not the typical style we use in other places.

Reply:
You are right it's not needed, dropped